### DISCUSSION TITLE
How to use secure routing in Flutter. ?

### DISCUSSION THREAD

**Discussion by aldodevv** [community/community#68972] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

In this case I need separate between save route and public route ??

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
